Assets/HoloToolkit/Input/Prefabs/MixedRealityCameraParent.prefab

```diff
@@ -667,6 +667,8 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   FloorQuad: {fileID: 1520281459093612, guid: 08d25ccfbbc595b4a820c7edd551d095, type: 2}
   boundaryHeight: 10
+  opaqueTrackingSpaceType: 1
```

Comment:
Why don't we just set it in MixedRealityCameraManager then? Instead of BoundaryManager?

Reply:
TrackingSpaceType isn't a camera setting, and boundary and tracking space type are strongly linked (and this was already being set in BoundaryManager; it just wasn't configurable until this change).

Reply:
How can tracking not be related to camera?